packages/ramp-core/src/fixtures/gazebo/p1-screen-1.vue

```diff
@@ -37,33 +37,34 @@
 </template>
 
 <script lang="ts">
-import { Vue } from 'vue-property-decorator';
-import { Sync } from 'vuex-pathify';
-
-import { PanelConfigRoute } from '@/store/modules/panel';
+import { defineComponent } from 'vue';
+import { sync } from '@/store/pathify-helper';
 import { PanelInstance } from '../../api';
 
-export default class GazeboP1Screen1V extends Vue {
-    // ❌ this is a horrible way, don't do that! this is directly tapping into the store and two-way binds `route` property fro a specific panel
-    // this will work, but all possible interactions should go through the API, because the store implementation might change and this will break
-    // 👇
-    @Sync('panel/items@p1.route') route!: PanelConfigRoute;
-
-    // ❌ also don't do this for the reasons above 👇
-    @Sync('panel/pinned') pinned!: PanelInstance | null;
-
-    url: string =
-        'https://i2.wp.com/freepngimages.com/wp-content/uploads/2017/08/wooden-garden-gazebo.png?w=860';
-
-    pinPanel(): void {
-        // this is fine, but the name of the panel is hardcoded there, so you wouldn't need to update it if it ever changes
-        const panel = this.$store.get<PanelInstance>(`panel/items@p1`)!;
-        this.pinned =
-            this.pinned === null || (this.pinned && this.pinned.id) !== 'p1'
-                ? panel
-                : null;
+export default defineComponent({
+    name: 'GazeboP1Screen1V',
+    data() {
+        return {
+            route: sync('panel/items@p1.route'),
+            pinned: sync('panel/pinned'),
+            url:
+                'https://i2.wp.com/freepngimages.com/wp-content/uploads/2017/08/wooden-garden-gazebo.png?w=860'
+        };
+    },
+    comnputed: {
```

Comment:
```suggestion
    computed: {
```
spelling :(

Reply:
Surprised it still worked 🤔

Donethanks